gensrc/thrift/BackendService.thrift

```diff
@@ -134,4 +159,6 @@ service BackendService {
     // release the context resource associated with the context_id
     DorisExternalService.TScanCloseResult close_scanner(1: DorisExternalService.TScanCloseParams params);
 
+    TStreamLoadRecordResult get_stream_load_record(1: string params);
```

Comment:
Rename `params` to a meaningful name, maybe `last_record_time`?
But I think we'd better use a timestamp (which is in Long Integer type) to pass this param to decouple the implementation between RPC interface and the implementation in BE.

Reply:
> Rename `params` to a meaningful name, maybe `last_record_time`?
> But I think we'd better use a timestamp (which is in Long Integer type) to pass this param to decouple the implementation between RPC interface and the implementation in BE.

Thank you.